Web/starter_project_iconics/components/common/tagComponent.tsx

```diff
@@ -0,0 +1,23 @@
+import React from "react"
+
+interface props {
+    onClick: ()=>void,
+    child:string,
+    width?: string
+}
+const TagComponent: React.FC<props> = ({child, onClick, width}) => {
```

Comment:
I am assuming you expect multiple people to use the tag component (since you put it inside the `common` folder)
so i would really recommend a description of what it is used for and how people should use it (cause otherwise they will not understand it would work for them and create their own)

basically justify that it should be a common component and other people should use it too (otherwise it should be in a folder that corresponds to the feature it belongs to)